config/initializers/qualifications/gcse.rb

```diff
@@ -0,0 +1,341 @@
+GCSE_SUBJECTS = [
+  'Accounting',
+  'Additional Applied Science',
+  'Additional Mathematics',
+  'Additional Science',
+  'Additional Science (Applied)',
+  'Additional Science (General)',
+  'Additional Science (Modular)',
+  'Additional Science A',
+  'Additional Science B',
+  'Agriculture and Land Use',
+  'Ancient History',
+  'Ancient History (Short Course)',
```

Comment:
Can we replace `Short Course` with `Short course` (lowercase ‘c’). This then follows the casing we’re using for ‘Double award’.

Reply:
Have done